packages/vite/src/node/plugins/asset.ts

```diff
@@ -33,6 +33,9 @@ const emittedHashMap = new WeakMap<ResolvedConfig, Set<string>>()
 export function assetPlugin(config: ResolvedConfig): Plugin {
   // assetHashToFilenameMap initialization in buildStart causes getAssetFilename to return undefined
   assetHashToFilenameMap.set(config, new Map())
+
+  // add own dictionary by directly assigning mrmine
+  mrmime.mimes['ico'] = 'image'
```

Comment:
I think we should open an issue or PR to https://github.com/lukeed/mrmime first? @lukeed may be able to do a release quick enough so we don't need to patch.

Reply:
https://github.com/lukeed/mrmime/issues/3

> That was an intentional part of the pkg design.

Reply:
@bluwy added 🙈

Reply:
ico is officially the `image/vnd.microsoft.icon` mime type and also has had image/x-icon recognized, but both are vendor-specific and/or experimental so mrmime does not include them. The image or image/icon solutions are all userland names and not officially true

Reply:
Thanks for the reference @lukeed 👍🏼
IIUC, it would be better to use `image/x-icon` instead of `image` here

Reply:
Perhaps ya, but mrmime only contains the official mimes. Everything else is customizable